### PR TITLE
Temperature Index Model

### PR DESCRIPTION
**Pull request by dfrisinghelli**
Added a new notebook about temperature index models.

### CODE REVIEW

File: notebooks/oggm-edu/mass_balance_gradients.ipynb [hunk 1]
```diff
@@ -546,7 +546,7 @@
    "metadata": {},
    "source": [
     "## What's next?\n",
-    "[Back to table of contents](welcome.ipynb)"
+    "[Back to table of contents](welcome.ipynb) or [notebook: Temperarute index models](temperature_index_model.ipynb)"
```
Comment (fmaussion) [inline notebooks/oggm-edu/mass_balance_gradients.ipynb]: Typo in temperature

File: notebooks/oggm-edu/flowline_model.ipynb [hunk 2]
```diff
@@ -27,26 +27,35 @@
...
+    "import oggm_edu as edu"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# import modules, constants and set plotting defaults\n",
```
Comment (fmaussion) [inline notebooks/oggm-edu/flowline_model.ipynb]: What's the purpose of the changes in `flowline_model.ipynb` ?

File: notebooks/oggm-edu/getting_started_with_notebooks.ipynb [hunk 3]
```diff
@@ -361,5 +361,5 @@
   }
  },
  "nbformat": 4,
- "nbformat_minor": 1
+ "nbformat_minor": 2
```
Comment (fmaussion) [inline notebooks/oggm-edu/getting_started_with_notebooks.ipynb]: Next time, try to commit only the files which have notable content change (I know that notebooks are messy to handle with Github...)

File: notebooks/oggm-edu/flowline_model.ipynb (hunk 2 above)
Comment (dfrisinghelli) [reply notebooks/oggm-edu/flowline_model.ipynb]: I created a new function to not repeat code and I improved some comments as I did the notebook myself.

Comment (fmaussion) [reply notebooks/oggm-edu/flowline_model.ipynb]: OK! I just wanted to know

File: notebooks/oggm-edu/mass_balance_gradients.ipynb (hunk 1 above)
Comment (dfrisinghelli) [reply notebooks/oggm-edu/mass_balance_gradients.ipynb]: Fixed.

**Comment by fmaussion** [comment]
Thanks!
